packages/gatsby-cli/src/structured-errors/error-map.ts

```diff
@@ -536,6 +536,20 @@ const errors = {
     level: Level.ERROR,
     docsUrl: `https://www.gatsbyjs.org/docs/gatsby-cli/#new`,
   },
+  "11614": {
+    text: ({
+      path,
+      filePath,
+      line,
+      column,
+    }): string => `The path "${path}" errored during SSR.
+
+    Edit its component ${filePath}:${line}:${column} to resolve the error.
+
+    See this docs page for more information https://gatsby.dev/debug-html`,
```

Comment:
```suggestion
    Edit its component ${filePath}:${line}:${column} to resolve the error.`,
```

As we already pass a `docsUrl` ink will already print a note to look at the docs :)

Reply:
it wasn't working unfortunately — it looks like the other keys on error objects aren't copied over in here https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/src/reporter/reporter.ts